src/main/java/pl/widulinski/scrapp/webDataToScrap/DataToScrapRepository.java

```diff
@@ -0,0 +1,18 @@
+package pl.widulinski.scrapp.webDataToScrap;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import pl.widulinski.scrapp.enums.Categories;
+
+import java.util.List;
+
+
+@Repository
```

Comment:
Spring chyba wie że to jest repozytorium, mozesz sprobwac skasować

Reply:
zrobione